packages/cardano-services/src/Program/services/postgres.ts

```diff
@@ -150,6 +150,7 @@ export const getPool = async (
   if (options?.postgresConnectionStringDbSync) {
     const pool = new Pool({
       connectionString: options.postgresConnectionStringDbSync,
+      connectionTimeoutMillis: 1000 * 60,
```

Comment:
Also consider adding `query_timeout`

Reply:
50ed4a64ca2ee723c6ee16e2393df164ce2ef3bb - also added missing configuration in other `Pool` constructor invocations